filter/O4DOIXmlFilter.php

```diff
@@ -201,20 +213,17 @@ function createSerialWorkNode($doc, $journalLocalePrecedence) {
 
 	/**
 	 * Create a title node.
-	 * @param $doc DOMDocument
-	 * @param $locale string e.g. 'en'
-	 * @param $localizedTitle string
-	 * @param $titleType string One of the O4DOI_TITLE_TYPE_* constants.
-	 * @param $localizedSubtitle string optional
-	 * @return DOMElement
+	 * @param string $titleType O4DOI_TITLE_TYPE_*
 	 */
-	function createTitleNode($doc, $locale, $localizedTitle, $titleType, $localizedSubtitle = null) {
+	function createTitleNode( DOMDocument$doc, string $locale, string $localizedTitle, string $titleType, string $localizedSubtitle = null): DOMElement
```

Comment:
There's an additional space before `DOMDocument` and a missing space between that and `$doc`. Doesn't appear to be causing an issue, though.